src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -2295,7 +2297,8 @@ void Hole::readCutDefinitions()
         for (const auto& f : files) {
             if (f.extension() == "json") {
                 try {
-                    std::ifstream input(f.filePath());
+                    Base::FileInfo fi(f.filePath());

```

Comment:
`f` already is a FileInfo object. So, no need to create a tmp. object.

Reply:
Thanks, [74f7154](https://github.com/FreeCAD/FreeCAD/pull/7383/commits/74f715431e534134984abd331e4586b981bc56af) is this correct now?